### PR TITLE
remove some unused resources

### PR DESCRIPTION
**Pull request by moving-bits**
taken from the last lint analysis

### CODE REVIEW

**Comment by moving-bits** [comment]
@mucek4 
Build on c-geo5 timed out + has an error message "Unknown API Level". Can you have a look at it? Thanks.

```
09:44:07  > Task :main:installBasicDebug
09:44:07  [PropertyFetcher]: ShellCommandUnresponsiveException getting properties for device emulator-5554: null
09:44:10  [PropertyFetcher]: ShellCommandUnresponsiveException getting properties for device emulator-5554: null
09:44:10  Skipping device 'android-23(AVD)' for 'main:basic-debug': Unknown API Level
09:44:10  
09:44:10  > Task :main:installBasicDebug FAILED
09:44:12  
[...]
09:44:32  Build timed out (after 25 minutes). Marking the build as aborted.
```

**Comment by moving-bits** [comment]
retest this please

**Comment by fm-sys** [comment]
> taken from the last lint analysis

Btw, Android Studio also has a feature to spot and remove all unused resources automatically...

**Comment by moving-bits** [comment]
> > taken from the last lint analysis
> 
> Btw, Android Studio also has a feature to spot and remove all unused resources automatically...

Thanks, I'll try to remember...
In this case I came from checking the build log and stumbled upon the lint result.

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -246,8 +246,6 @@
     <string name="warn_log_text_fill">Please input some text for your log.</string>
     <string name="warn_load_images">c:geo failed to load images.</string>
     <string name="warn_invalid_mapfile">The selected map file is not a valid mapsforge map file or uses an unsupported version.\n(c:geo currently supports v3 to v5 / v3 only if you are using \"Old Mapfsforge V3\" setting.)\n\nOffline maps are not available.</string>
-    <string name="warn_nonexistant_mapfile">The selected map file does not exist.\nOffline maps are not available.</string>
-    <string name="warn_rendertheme_missing">Map theme not found.</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:250]: I wonder, why these error messages are no longer in use. As I swap my SD from time to time I saw them in the past.

**Comment by mucek4** [comment]
> 
> 
> @mucek4
> Build on c-geo5 timed out + has an error message "Unknown API Level". Can you have a look at it? Thanks.
> 
> ```
> 09:44:07  > Task :main:installBasicDebug
> 09:44:07  [PropertyFetcher]: ShellCommandUnresponsiveException getting properties for device emulator-5554: null
> 09:44:10  [PropertyFetcher]: ShellCommandUnresponsiveException getting properties for device emulator-5554: null
> 09:44:10  Skipping device 'android-23(AVD)' for 'main:basic-debug': Unknown API Level
> 09:44:10  
> 09:44:10  > Task :main:installBasicDebug FAILED
> 09:44:12  
> [...]
> 09:44:32  Build timed out (after 25 minutes). Marking the build as aborted.
> ```

I have restarted both docker containers, but it seems it does not help. I am however unable to create new ones, as emulator does not start.

**Pull request merged**